.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+on:
+  - pull_request
+  - push
+
+name: CI
+
+jobs:
+  php-lint:
+    name: PHP linter
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version:
+          - 7.2
+          - 7.3
+          - 7.4
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Run PHP lint
+      run: composer run-script ci:php:lint
```

Comment:
Why not the shorter form?

`composer ci:php:lint`

Reply:
true… I think that's a leftover from the GitHub action template I had a look at.